README.md

```diff
@@ -119,7 +119,88 @@ Uncaught exception in angular expressions will now be reported to Stackdriver Er
 
 If you wish, you can manually delegate exceptions, e.g. `try { ... } catch(e) { $exceptionHandler(e); }` or simply `$exceptionHandler('Something broke!');`.
 
+## Setup for ReactJS
 
+Follow the general instructions denoted in _Setup for JavaScript_ to load and initialize the library.  
+
+There is nothing specific that needs to be done with React, other than making sure to initialize the library in your root entry point(typically `index.js`).
+
+## Best Practices
+
+### Usage with Webpack
```

Comment:
I'd suggest to be more specific in this title. Something like 'Only report in prod environment with Webpack'